src/commands.cpp

```diff
@@ -3673,16 +3673,10 @@ void CommandGetUserData::procresult()
                         || (m == BIZ_MODE_UNKNOWN))  // master flag not received or invalid
                 {
                     LOG_err << "GetUserData: invalid business status / account mode";
+                    client->app->notify_business_status(BIZ_STATUS_UNKNOWN);
```

Comment:
Check with the apps teams the expected behavior for an unknown business status.

Also, if an unknown status is notified, it must update the client's state: `MegaClient::mBizStatus` and the related class members as well.

Perhaps, a possible workaround would be fallback to `BIZ_STATUS_INACTIVE` and send an event to the stats server to record the unexpected data received from API.

Reply:
Agreed with Webclient that the fallback status in case of invalid data in the `ug` response for the `b` element is `BIZ_STATUS_EXPIRED`, since that status will keep the business level for the account, but will block most of the operations.